### PR TITLE
Create main.yml

### PR DESCRIPTION
**Pull request by sschacko**
Please review and if it's a good start, merge the feature-item to FTR-actions as per the instructions of the P4 README.md

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    container: openjdk
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      # Runs a set of commands using the runners shell
+      - name: Run a multi-line script
+        run: |
+          javac -cp "../Projects/P3/tests/junit-4.10.jar:." *.java
```
Comment (striker4150) [inline .github/workflows/main.yml:21]: Keep in mind that the entire repository is checked out, so you need to compile the `.java` files in the **Projects/P3** directory.

**Review by striker4150 — CHANGES_REQUESTED** [review]
In addition to fixing the `javac` command, please remove the unnecessary `.github/workflows/main.yml` in the `Projects/P3/tests` directory. According to [this](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#about-yaml-syntax-for-workflows), workflows can only be put in the root directory of the repository.

File: .github/workflows/main.yml (hunk 1 above)
Comment (sschacko) [reply .github/workflows/main.yml]: I think I followed what you were saying, but it still had build errors

Comment (striker4150) [reply .github/workflows/main.yml]: See slack thread

**Review by striker4150 — DISMISSED** [review]
Please delete the .class file.

**Review by striker4150 — APPROVED** [review]

**Pull request merged**